packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/modal/EditorMain.vue

```diff
@@ -137,7 +140,18 @@ defineSlots<{
 
 const { modalOpen, undo, redo, canUndo, canRedo, load } = useEditorStore()
 
-function selectExample(example: string) {
+const enableDatakitM2 = inject<boolean>(FEATURE_FLAGS.DATAKIT_M2, false)
+
+const M2_EXAMPLES = ['vault', 'cache']
```

Comment:
nit: Safer type, but the current LGTM as this is temporary 

```suggestion
const M2_EXAMPLES: Array<keyof typeof examples> = ['vault', 'cache']
```